### DISCUSSION TITLE
What to do with the Freenode IRC channel?

### DISCUSSION THREAD

**Discussion by refi64** [hylang/hy#2075] [General]
Hey there, long time no see! I'm checking in from the abyss, because it turns out Freenode is kind of [undergoing some major issues](https://gist.github.com/joepie91/df80d8d36cd9d1bde46ba018af497409), and almost every channel I've been on has either moved their IRC channel to Libera.chat or OFTC, or switched altogether to another protocol (Matrix being a common choice).

Right now, there's only a few channels left on Freenode, with #hy being one of them. I have no idea who has ops there, nor who is usually active there, but it might be worth finally considering moving off of IRC or at least to a server that isn't actively, uhh, taking over channels? 😅

**Comment by allison-casey** [comment]
Pretty sure the current plan is to just get rid of it in favor of GitHub discussions. See #1778

**Reply by refi64** [reply]
Right I've been following that convo though I didn't add anything to it for a while, originally I had just assumed Discussions would be more of a Groups alternative since the type of conversation it has is a lot less interactive/chat-style and more forum-style...but did we ever use IRC for *that* much anyway? During the peak of my project involvement I don't recall ever using it much.

**Reply by allison-casey** [reply]
I don't think i've ever even opened it up after initially realizing it existed :woman_shrugging: 

I was thinking of starting an unofficial community discord once we hit 1.0 just as a place for people to hang out and share what they're doing with Hy to scratch that chat based itch that discussions doesn't really cover.

**Comment by brandonwillard** [comment]
I'm in favor of moving to Matrix.

**Reply by jgarte** [reply]
It would be amazing to have a hy matrix channel!

Is there a way to backup or take logs of github discussions in an automated way? Via the api?

I think there are some important points made in this video regarding proprietary services like github: https://diode.zone/videos/watch/2ac71859-4ded-44e9-81fc-b4a9c1af4536?start=14s

Free Software Needs Free Tools

**Reply by alphapapa** [reply]
A long time ago I made the channel `#hylang:matrix.org`.  So far no one has ever joined it.  Anyway, I'd be glad to turn it over to whoever should be in charge of it.

**Reply by alphapapa** [reply]
Haha, I just realized that I had left the room on invite-only all this time.  That might explain why no one has ever joined.  That's fixed now.
